packages/matter.js/src/device/PairedNode.ts

```diff
@@ -395,47 +395,70 @@ export class PairedNode {
     }
 
     /**
-     * Force a reconnection to the device. This method is mainly used internally to reconnect after the active session
+     * Trigger a reconnection to the device. This method is non-blocking and will return immediately.
+     * The reconnection happens in the background. Please monitor the state of the node to see if the
+     * reconnection was successful.
+     */
+    triggerReconnect() {
+        if (this.#reconnectionInProgress || this.#remoteInitializationInProgress) {
+            logger.debug("Reconnection or initialization already in progress ...");
```

Comment:
Maybe mention why it's logging this, e.g. `` `Ignoring reconnect request because ${this.#remoteInitializationInProgress ? "init" : "reconnect"} already underway` ``

Reply:
good idea .. and I also changed logging to info in the trigger method